src/main.cpp

```diff
@@ -248,12 +248,12 @@ struct Speeds PID(struct Speeds targetSpeeds, struct Speeds commandSpeeds) {
     display.printf("heading: %2.2f", orientationReading.x);
     display.display();
     commandSpeeds.left += steeringCorrection;
-    commandSpeeds.right += steeringCorrection;
+    commandSpeeds.right -= steeringCorrection;
```

Comment:
d'oh

Reply:
this was a legit change (both versions were/are correct), since we changed which direction the motors turned